lib/LedgerSMB/Template/DB.pm

```diff
@@ -131,12 +131,15 @@ does not call this without carefully whitelisting values.
 
 sub get_from_file {
     my ($package, $path, $language_code) = @_;
-    my $fname = $path;
+    my $fname;
     if ($path =~ m|/.*:| ){
        die 'Cannot run on NTFS alternate data stream!';
     }
-    $path =~ m|(.*)/([^/]+)$|;
-    $fname = $2;
+    if ( $path =~ m|(.*)/([^/]+)$| ) {
+        $fname = $2;
+    }  else {
+        $fname = $path;
+    }
```

Comment:
@ehuelsmann there is a change to the logic here. __*I THINK*__ it's a good change, but can you confirm there are no unwanted side affects please.

Original Logic
- fname = $path
- match then fname = $2
- the result is fname should end up as = $2, or an undefined value that may have already been in $2 from the previous match

New Logic
- If match fname = $2
- if not match fname = $path
- result is the raw path could end up in fname. Is this desirable?

What I infer from the original logic is we probably want the else clause to set $fname = '' to make the path relative as the most likely behaviour would be $2 = undef